src/app/settings/settings.component.ts

```diff
@@ -34,18 +34,10 @@ export class SettingsComponent implements OnInit {
     //update the ui
     this.selectedTheme = event.target.value;
 
-    if (this.selectedTheme == "light") {
-      this.themeService.setTheme("light");
-      this.whichTheme = "light";
-      localStorage.setItem("theme", "light");
-    } else if (this.selectedTheme == "dark") {
-      this.themeService.setTheme("dark");
-      this.whichTheme = "dark";
-      localStorage.setItem("theme", "dark");
-    } else if (this.selectedTheme == "iceydark") {
-      this.themeService.setTheme("iceydark");
-      this.whichTheme = "iceydark";
-      localStorage.setItem("theme", "iceydark");
+    if (this.selectedTheme == this.selectedTheme) {
```

Comment:
Also you don't need this `if (this.selectedTheme == this.selectedTheme) {` because that will always resolve to `true` anyways! 😉

Reply:
lol thank you for the logic checkup today @connerdouglass 😆 fixes incoming!

Reply:
Sure thing @carsenk. You're crushing it with these frontend improvements!

Reply:
Gotta love typescript!